web-common/src/features/dashboards/dimension-table/DimensionValueHeader.svelte

```diff
@@ -38,11 +40,14 @@
 
   $: atLeastOneSelected = !!selectedIndex?.length;
 
-  const getCellProps = (row) => {
-    const value = rows[row.index][column.name];
+  const getCellProps = (row: VirtualItem) => {
+    const value = rows[row.index]?.[column.name];
```

Comment:
I believe this invalid array access was the actual cause of the crash, everything else is just cleanup to make it clearer what data we're dealing with here